components/Receipt.js

```diff
@@ -402,7 +404,9 @@ export class Receipt extends React.Component {
                   <Flex justifyContent="space-between">
                     <Box>
                       <H2 fontSize="16px" lineHeight="18px">
-                        {receipt.template?.title || (
+                        {receipt.template?.title || receipt.isRefundOnly ? (
+                          <FormattedMessage defaultMessage="Payment refund" />
+                        ) : (
```

Comment:
@Betree : I think includes a small regression. If we put a title for the template, the associated receipt has the heading "Payment refund". An example in staging I created is, https://staging.opencollective.com/dashboard/opencollective/host-transactions?sort%5Bfield%5D=CREATED_AT&sort%5Bdirection%5D=DESC&searchTerm=233804&kind=CONTRIBUTION